examples/plugins/traffic-control/tc.go

```diff
@@ -31,18 +37,40 @@ func DoTrafficControl(pid int, latency string, pktLoss string) error {
 
 	}
 	cmd := split("tc qdisc change dev eth0 root handle 1: netem")
-	if latency != "" {
+	// TODO @alepuccetti: refactor this code
+	if latency == "" {
```

Comment:
What about separate functions for adjusting the latency and packet loss, wich some shared helper funcs? Instead of detecting based on args...